app/javascript/packages/document-capture/components/document-capture.tsx

```diff
@@ -40,6 +40,52 @@ function DocumentCapture({ onStepChange = () => {} }: DocumentCaptureProps) {
   const { inPersonFullAddressEntryEnabled, inPersonURL, skipDocAuth, skipDocAuthFromHandoff } =
     useContext(InPersonContext);
   const appName = getConfigValue('appName');
+  const inPersonLocationPostOfficeSearchForm = inPersonFullAddressEntryEnabled
+    ? InPersonLocationFullAddressEntryPostOfficeSearchStep
+    : InPersonLocationPostOfficeSearchStep;
+
+  // Define different states to be used in human readable array declaration
+  const documentFormStep: FormStep = {
+    name: 'documents',
+    form: DocumentsStep,
+    title: t('doc_auth.headings.document_capture'),
+  };
+  const reviewFormStep: FormStep = {
+    name: 'review',
+    form:
+      submissionError instanceof UploadFormEntriesError
+        ? withProps({
+            remainingSubmitAttempts: submissionError.remainingSubmitAttempts,
+            isResultCodeInvalid: submissionError.isResultCodeInvalid,
+            isFailedResult: submissionError.isFailedResult,
+            isFailedDocType: submissionError.isFailedDocType,
+            isFailedSelfie: submissionError.isFailedSelfie,
+            isFailedSelfieLivenessOrQuality:
+              submissionError.selfieNotLive || submissionError.selfieNotGoodQuality,
+            captureHints: submissionError.hints,
+            pii: submissionError.pii,
+            failedImageFingerprints: submissionError.failed_image_fingerprints,
+          })(ReviewIssuesStep)
+        : ReviewIssuesStep,
+    title: t('errors.doc_auth.rate_limited_heading'),
+  };
+
+  // In Person Steps
+  const prepareFormStep: FormStep = {
+    name: 'prepare',
+    form: InPersonPrepareStep,
+    title: t('in_person_proofing.headings.prepare'),
+  };
+  const locationFormStep: FormStep = {
+    name: 'location',
+    form: inPersonLocationPostOfficeSearchForm,
+    title: t('in_person_proofing.headings.po_search.location'),
+  };
+  const hybridFormStep: FormStep = {
+    name: 'switch_back',
+    form: InPersonSwitchBackStep,
+    title: t('in_person_proofing.headings.switch_back'),
+  };
 
   useDidUpdateEffect(onStepChange, [stepName]);
```

Comment:
If possible, I'd suggest keeping the component hooks grouped together with the others at the top of the function, so that it's easy to find them in one spot.

Reply:
@aduth I believe they are all grouped together (lines 48 - 88), could you point me to where they are not grouped together?

Reply:
There are hooks on lines 34-41, and then separately hooks on line 90-95. My recommendation is to bump up those hooks on line 90-95 to be together with the ones on 34-41.

https://github.com/18F/identity-idp/blob/c86f7ed9a595a8721dab707d3f09790caa61676e/app/javascript/packages/document-capture/components/document-capture.tsx#L34-L41

https://github.com/18F/identity-idp/blob/c86f7ed9a595a8721dab707d3f09790caa61676e/app/javascript/packages/document-capture/components/document-capture.tsx#L90-L95